docs/resources/vault_secrets_app_iam_binding.md

```diff
@@ -0,0 +1,19 @@
+---
+page_title: "{{.Type}} {{.Name}} - {{.ProviderName}}"
+subcategory: "Cloud Platform"
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+~> **Note:** `hcp_vault_secrets_app_iam_binding` can only be used if the app has an existing policy and cannot be used in conjunction with
```

Comment:
This seems like a bug? From a user perspective it should just work to add the first binding.

Reply:
This is blocking from my perspective

Reply:
This has been fixed

Reply:
Should the text be updated then?